src/js/components/search/filters/location/LocationSearch.jsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * LocationSearch.jsx
+ * Created by Emily Gullo 11/28/2016
+ **/
+
+import React from 'react';
+import CountryType from './CountryType';
+import SelectedLocations from './SelectedLocations';
+import LocationListContainer from '../../../../containers/sharedContainers/LocationListContainer';
```

Comment:
Let's change this import to use the new path resolving set up a few sprints ago. You can specify paths relative to the `/src/js` directory if you ditch the leading `./` or `../`.